src-tool/Pact/Analyze/Check.hs

```diff
@@ -106,8 +106,14 @@ smtConfig :: SBV.SMTConfig
 smtConfig = SBV.z3
   { SBVI.allowQuantifiedQueries = True
   , SBVI.verbose = False -- set to True for debugging SMT formulas
+  , SBVI.transcript = Nothing -- Just "smt.transcript"
   }
 
+-- | Solver timeout in millis. Usually timeout is a bad sign
+-- but can always try larger values here.
+timeout :: Integer
+timeout = 10000
```

Comment:
@emilypi are we fine with increasing the z3 timeout from 1s to 10s?

Reply:
I am going to merge the PR in order to fix ci builds on master. We can address this question later.